### PR TITLE
extend data clean to handle ENSEMBLE data

### PR DESCRIPTION
**Pull request by youyifong**

### CODE REVIEW

**Comment by youyifong** [comment]
This PR also restricts multitesting adjustment for COVE to without ID80

File: cor_coxph/.Rprofile [hunk 1]
```diff
@@ -0,0 +1 @@
+source("renv/activate.R")
```
Comment (nhejazi) [inline cor_coxph/.Rprofile:1]: Make sure to delete this dotfile. It is automatically created by `renv` upon instantiation.

**Comment by youyifong** [comment]
Thanks!

File: _common.R [hunk 2]
```diff
@@ -100,6 +100,7 @@ assays_to_be_censored_at_uloq_cor <- c(
 # figure labels and titles for markers
 ###############################################################################
 has29 = "Day29" %in% times
+has57 = study_name_code=="COVE"
```
Comment (benkeser) [inline _common.R:103]: Why not `"Day57" %in% times`? Might generalize better when we need to handle AZ/Nova.

**Comment by benkeser** [comment]
A couple `pdf`'s in the PR, e.g., `cor_nonlinear/output/D57/marginalized_risks_gam_MockCOVE.pdf`. Please add to `gitignore` or delete.

**Comment by youyifong** [comment]
Good catch. Will update. There is an option to add all PDFs to ignore list.
Is that okay to do?

**Comment by benkeser** [comment]
> Is that okay to do?

I think some of the verification reports are saved as pdfs and are included in the repo (@thebioengineer can confirm). But otherwise, I'm good with ignoring all pdfs.

**Comment by youyifong** [comment]
has57 is the key in how I ended up extending data clean to different
trials, instead of splitting into trial specific cleaning scripts. And I
think that is why I define it this way. But as we keep extending, we may
find it makes more sense to define it based on the times variable.

**Comment by thebioengineer** [comment]
We should keep pdfs, though in 
.gitignore you can add an inverse to keep files that might normally be removed by the ignore

**Comment by youyifong** [comment]
That is a cool feature. I will follow prompt to add all PDFs to ignore list
next time then. Thanks!

File: config.yml [hunk 3]
```diff
@@ -37,26 +36,27 @@ janssen_pooled_real: &janssen_default_real
 janssen_na_real: &janssen_default_real
   <<: *janssen_default_real
   subset_variable: Region
-  subset_value: Northern America
+  subset_value: 0
```
Comment (benkeser) [inline config.yml:39]: Are you confident that this numeric labeling is stable? I get nervous about how factors/characters get cast to numerics, so just wanted to double check.

**Comment by youyifong** [comment]
The config package seems to be doing what we expect:

> config
$data_raw_dir
[1] "janssen"

$times
[1] "B"            "Day29"        "Delta29overB"

$data_in_file
[1] "COVID_ENSEMBLE_practicedata.csv"

$study_name
[1] "MockENSEMBLE"

$study_name_code
[1] "ENSEMBLE"

$assays
[1] "bindSpike" "bindRBD"

$subset_variable
[1] "Region"

$subset_value
[1] 1

attr(,"config")
[1] "janssen_la_mock"
attr(,"file")
[1] "D:\\gdrive\\Covid19\\correlates_reporting/config.yml"

> str(config$subset_value)
 int 1

**Comment by benkeser** [comment]
I mean do we have these labels for regions hard coded or are they determined on the fly in data processing?

**Comment by youyifong** [comment]
They are coded as integer in the raw data file:

> str(dat_proc$Region)
 int [1:43783] 1 0 0 1 0 1 1 1 0 2 ...

In the readme:

Region          = Major geographic region of the study site of enrollment
(0=Northern America, 1=Latin America,
                  2=Southern Africa, no missing values allowed). The names
"Northern America", "Southern Africa",
                  "Latin America" are used in the REGION1 variable in the
Janssen ADSL data set.  Northern America means U.S.
                  The Latin America region includes Argentina, Brazil,
Chile, Columbia, Mexico, Peru.

Conversion will happen in the step to make raw data I think.

**Pull request merged**